sdk/core/azure-core-amqp/src/samples/java/com/azure/core/amqp/models/AmqpAnnotatedMessageJavaDocCodeSamples.java

```diff
@@ -13,24 +15,26 @@
     public void checkBodyType() {
         AmqpAnnotatedMessage amqpAnnotatedMessage = null;
         // BEGIN: com.azure.core.amqp.models.AmqpBodyType.checkBodyType
-        // If client do not check `AmqpMessageBody.getBodyType()` and payload is not of type `AmqpMessageBodyType.DATA`,
-        // calling `getFirstData()` or `getData()` on `AmqpMessageBody` will throw Runtime exception.
-        // https://github.com/Azure/azure-sdk-for-java/issues/17614 : This issue tracks additional AMQP body type
-        // support in future.
-
-        byte[] payload = null;
+        Object amqpValue;
         AmqpMessageBodyType bodyType = amqpAnnotatedMessage.getBody().getBodyType();
+
         switch (bodyType) {
             case DATA:
-                payload = amqpAnnotatedMessage.getBody().getFirstData();
+                byte[] payload = amqpAnnotatedMessage.getBody().getFirstData();
+                System.out.println(new String(payload));
                 break;
             case SEQUENCE:
+                List<Object> sequenceData = amqpAnnotatedMessage.getBody().getSequence();
+                sequenceData.forEach(System.out::println);
+
+                break;
             case VALUE:
-                throw new RuntimeException("Body type not supported yet.");
+                amqpValue = amqpAnnotatedMessage.getBody().getValue();
+                System.out.println(amqpValue);
```

Comment:
remove this line?

Reply:
It is fine because we are outputting the value in same , so user can see it.